javaee/arquillian-example/pom.xml

```diff
@@ -97,8 +96,8 @@
         </dependency>
         <dependency>
             <groupId>fish.payara.arquillian</groupId>
-            <artifactId>arquillian-payara-server-4-embedded</artifactId>
-            <version>1.0.Beta3</version>
+            <artifactId>arquillian-payara-server-embedded</artifactId>
+            <version>2.0</version>
```

Comment:
Don't need to specify version, it's managed in the bom https://github.com/payara/Payara/blob/master/api/payara-bom/pom.xml

Reply:
then the root parent pom of the examples needs further work to pull in the bom as an inherited dependency

Reply:
In that case we'll consider it outside the scope of this change

Reply:
Sorry, just noticed the project. This has been moved to payara/payara